### DISCUSSION TITLE
Font sizes

### DISCUSSION THREAD

**Discussion by rom169** [marcokreeft87/room-card#273] [General]
I'm using latest version of room card with latest version of HA core.

Tried to change font sizes of room name an entity names in entities list with styles and card-mod without success.

Could you please provide an example on how to do it correctly?

**Comment by marcokreeft87** [comment]
What do you have so far?

Btw, this isnt a bug, please ask your question in discussions next time

**Comment by rom169** [comment]
Solved it by adding this to the card:
`
card_mod:
  style: |
    ha-card {
      --paper-item-icon-color: steelblue;
      color: rgb(224,224,224);
    }
    div.title.clickable {
      font-size: 14px !important; 
      font-weight: bold !important;
    }
    span {
      font-size: 12px !important; 
      font-weight: bold !important;
      color: rgb(155,155,155);
    }
`
This leads to a design quite close to mushroom cards.
Best regards

**Comment by hobili** [comment]
![Screenshot 2024-05-31 134351](https://github.com/marcokreeft87/room-card/assets/56883534/a1bd4ca9-c8f5-42e0-a676-bbe3b526c4c4)
Hi! 

with...

```
                    card_mod:
                      style: |
                        div.title.clickable { 
                          font-size: 20px !important;  
                        } 
```

I can change the size of the Card-Text on the left side - but what do I have to do to change the size of the Text of the info_entities ?

**Comment by marcokreeft87** [comment]
Try 
```
div.entities-info-row .entity  {

}
```

Btw tbf, the card isnt designed for bigger text in the info_entities section. Probably will work, but it might shift the layout

**Reply by hobili** [reply]
great! Thank you very much! ;-) 

yes, it's a little bit shifted but thats ok.

**Comment by petergCA** [comment]
@marcokreeft87 do you have any suggestions for making the entity names and icons larger? They are simply too small for the screen. Thanks!

**Reply by marcokreeft87** [reply]
@petergCA did you try the answer above?

**Reply by petergCA** [reply]
@marcokreeft87 ... Yes, that is for the header/entities row though and not the entities themselves. I've tried a number of things but all that happens is the space between the icon and text changes - I can't get the text or icon itself to change size (larger)

**Reply by marcokreeft87** [reply]
If that doesn't work, card-mod should work but I can't help you with that
